src/steps.js

```diff
@@ -101,21 +119,117 @@ const ipvHandOff = async (page) => {
   });
 };
 
-// Note only used by IPV canary
-const microclientUserInfo = async (page, email) => {
-  await synthetics.executeStep("Microclient user info", async () => {
-    await page.content();
+// Steps only used by Create Account canary
 
-    const userInfo = await page.evaluate(() => {
-      // eslint-disable-next-line no-undef
-      return JSON.parse(document.querySelector("body").innerText);
-    });
+const clickCreateAccount = async (page) => {
+  await synthetics.executeStep("Click create account", async () => {
+    await page.waitForSelector("#main-content #create-account-link");
+    await Promise.all([
+      page.waitForNavigation({
+        waitUntil: "networkidle0",
+      }),
+      page.click("#main-content #create-account-link"),
+    ]);
+    await validateUrlContains("enter-email-create", page);
+  });
+};
 
-    const hasReachedUserInfo = userInfo.email === email;
+const submitEmailCreate = async (page) => {
+  await synthetics.executeStep("Click continue", async () => {
+    await page.waitForSelector(selectors.submitFormButton);
+    await Promise.all([
+      page.waitForNavigation({
+        waitUntil: "networkidle0",
+      }),
+      page.click(selectors.submitFormButton),
+    ]);
+  });
+  await validateUrlContains("check-your-email", page);
+};
 
-    if (!hasReachedUserInfo) {
-      throw new Error(`Failed at Microclient user info step`);
-    }
+const createPassword = async (page, password) => {
+  await synthetics.executeStep("Create password", async () => {
+    await page.waitForSelector(selectors.passwordInput);
+    await page.type(selectors.passwordInput, password);
+    await page.type(".govuk-grid-row #confirm-password", password);
+  });
+};
+
+const submitCreatePassword = async (page) => {
+  await synthetics.executeStep("Click continue", async () => {
+    await page.waitForSelector(selectors.submitFormButton);
+    await Promise.all([
+      page.waitForNavigation({
+        waitUntil: "networkidle0",
+      }),
+      page.click(selectors.submitFormButton),
+    ]);
+    await validateUrlContains("get-security-codes", page);
+  });
+};
+
+const chooseSMSForSecurityCodes = async (page) => {
+  await synthetics.executeStep("Choose how to get security codes", async () => {
+    await page.waitForSelector(".govuk-grid-row #mfaOptions");
+    await page.click(".govuk-grid-row #mfaOptions");
+  });
+};
+
+const submitSecurityCodesChoice = async (page) => {
+  await synthetics.executeStep("Click continue", async () => {
+    await page.waitForSelector(selectors.submitFormButton);
+    await Promise.all([
+      page.waitForNavigation({
+        waitUntil: "networkidle0",
+      }),
+      page.click(selectors.submitFormButton),
+    ]);
```

Comment:
We do a lot of "Wait for page load AND do other thing" - could we refactor out a function that encapsulates this behaviour?

Otherwise, looks great!